tasks/main.yml

```diff
@@ -37,24 +37,30 @@
   tags:
       - xpack
 
-- include: elasticsearch-templates.yml
+- include: elasticsearch-template-copy.yml
   when: es_templates
   tags:
       - templates
 
 - meta: flush_handlers
 
+- include: elasticsearch-template-insert.yml
```

Comment:
Notice we flush the handlers before doing the insert. This is to ensure a restart occurs if needed. Suppose the user makes a config change which impacts the ability to insert templates - the first one that springs to mind is turning off security. Admittedly this is a niche case.